frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -2108,7 +2108,7 @@ function ReaderHighlight:onHighlightPress()
                 self:onStopHighlightIndicator(true) -- need_clear_selection=true
             end
         else
-            self:onHoldRelease(nil, self:_createHighlightGesture("hold_release"))
+            self:onHoldRelease()
```

Comment:
Or more probably, this is just building and passing arguments as we do to most onTouchGestureHandler, passing `(arg, ges)`.
It just happens that our own  `ReadHighlight:onHoldRelease()` in that file does not need them, so it was declared without them.
I think the proper way would be to just declare /define `ReadHighlight:onHoldRelease(arg, ges)` - as most other :onSomething in readerhighlight.lua - even if the implementation does not use them.

Reply:
Indeed, I'd also argue it's better than the alternative. But your suggestion introduces an unused local variable; something linters always complain in such event handler type situations to my annoyance. :-)

Reply:
Recent luacheck versions will grok if if prefixed by an underscore, IIRC (e.g., `func(_foo, _bar)`).

Reply:
@poire-z You are right. It is creating fake events following touch call protocol `(arg, ges)`. Lua function handler can declare ignore arguments. But changes to `self:onHoldRelease()` is not a good idea.

Reply:
> But changes to `self:onHoldRelease()` is not a good idea

The call in question or the implementation?

(i.e., is that just a NAK on that commit?)

Reply:
I mean the call: removing arguments break event handler function protocol. It may cause unexpected issues in future:
* If there is a chance that some one will override `onHoldRelease` and he needs arguments, then the application will crash due to null pointer exception.
* The event protocol is solid: there are always two arguments passed to event handler. Widget author will be surprised that it is not always true. It will make event handler function need more code to handle such unexpected behavior.

Follow the protocol make life easy.